src/common/util/charades-words.js

```diff
@@ -0,0 +1,62 @@
+const https = require('https');
```

Comment:
this file shouldn't exist, i just copied it over from the existing thing that downloads a dictionary because i needed it fast

ideally, the whole "download a thing and extract info from it for me to reference later" thing should be abstracted out to the point where the dictionary and charades word functions are basically the same thing referencing a different source URL.

Reply:
This will be a different PR